File: 2025-2/sesion-de-clase-02/.NET/Ejercicio01/Programa.cs
using System;

namespace Ejercicio01 {
    public class Programa {
        public static void Main(string[] args) {
            Alumno persona = (Alumno)new Alumno.AlumnoBuilder()
                .ConActividad(TipoActividad.Baile)
                .ConNombre("Juan")
                .ConEdad(22)
                .Build();

            //Console.WriteLine(persona);
            persona.Imprimir();

            IPracticable practicable = (IPracticable)new Alumno.AlumnoBuilder()
                .ConActividad(TipoActividad.Deporte)
                .ConNombre("Ana")
                .ConEdad(21)
                .Build();

            practicable.Practicar();
        }
    }
}

File: 2025-2/sesion-de-clase-02/Java/ejercicio01/src/main/java/pe/edu/pucp/inf30/ejercicio01/Programa.java
package pe.edu.pucp.inf30.ejercicio01;

/**
 *
 * @author eric
 */
public class Programa {
    public static void main(String[] args) {
        Persona persona = new Alumno.AlumnoBuilder()
                .nombre("Juan")
                .edad(22)
                .build(); 
//        System.out.println(persona);
        persona.imprimir();

        IPracticable alumno = new Alumno.AlumnoBuilder()
                .nombre("Ana")
                .edad(21)
                .actividad(TipoActividad.Musica)
                .build();
        alumno.practicar();
    }
}
